tools/sst_dump_tool.cc

```diff
@@ -158,7 +158,8 @@ Status SstFileDumper::DumpTable(const std::string& out_filename) {
 }
 
 uint64_t SstFileDumper::CalculateCompressedTableSize(
-    const TableBuilderOptions& tb_options, size_t block_size) {
+    const TableBuilderOptions& tb_options, size_t block_size,
+    uint64_t& num_data_blocks) {
```

Comment:
We follow Google C++ Style and it bans using reference as output argument: https://google.github.io/styleguide/cppguide.html#Reference_Arguments